doc/releases/changelog-0.9.0.md

```diff
@@ -166,204 +180,221 @@
-  argument to `qjit`.
-  [(#1158)](https://github.com/PennyLaneAI/catalyst/pull/1158)
+  Note that `catalyst-cli` is only available when Catalyst is built from source, and is not included
+  when installing Catalyst via pip or from wheels.
+
+* Experimental integration of the PennyLane capture module is available. It currently only supports
+  quantum gates, without control flow.
+  [(#1109)](https://github.com/PennyLaneAI/catalyst/pull/1109)
+
+  To trigger the PennyLane pipeline for capturing the program as a Jaxpr, simply set
+  `experimental_capture=True` in the qjit decorator.
 
   ```python
-  @qjit(static_argnames="y")
-  def f(x, y):
-    if y < 10:  # y needs to be marked as static since its concrete boolean value is needed
-        return x + y
-
-  @qjit(static_argnames=["x","y"])
-  def g(x, y):
-    if x < 10 and y < 10:
-        return x + y
-
-  res_f = f(1, 2)
-  res_g = g(3, 4)
-  print(res_f, res_g)
-  ```
+  import pennylane as qml
+  from catalyst import qjit
 
-  ```pycon
-  3 7
+  dev = qml.device("lightning.qubit", wires=1)
+
+  @qjit(experimental_capture=True)
+  @qml.qnode(dev)
+  def circuit():
+      qml.Hadamard(0)
+      qml.CNOT([0, 1])
+      return qml.expval(qml.Z(0))
   ```
 
 <h3>Improvements</h3>
 
-* Adjoint canonicalization is now available in MLIR for `CustomOp` and `MultiRZOp`. It can be used
-  with the `--canonicalize` pass in `quantum-opt`.
-  [(#1205)](https://github.com/PennyLaneAI/catalyst/pull/1205)
+* Multiple `qml.sample` calls can now be returned from the same program, and can be structured using
+  Python containers. For example, a program can return a dictionary of the form
+  `return {"first": qml.sample(), "second": qml.sample()}`.
+  [(#1051)](https://github.com/PennyLaneAI/catalyst/pull/1051)
 
-* Implement a Catalyst runtime plugin that mocks out all functions in the QuantumDevice interface.
+* Catalyst now ships with `null.qubit`, a Catalyst runtime plugin that mocks out all functions in
+  the QuantumDevice interface. This device is provided as a convenience for testing and benchmarking
+  purposes.
   [(#1179)](https://github.com/PennyLaneAI/catalyst/pull/1179)
 
-* Scalar tensors are eliminated from control flow operations in the program, and are replaced with
-  bare scalars instead. This improves compilation time and memory usage at runtime by avoiding heap
-  allocations and reducing the amount of instructions.
-  [(#1075)](https://github.com/PennyLaneAI/catalyst/pull/1075)
+  ```python
+  qml.device("null.qubit", wires=1)
 
-  A new MLIR pass `detensorize-scf` is added that works in conjuction with the existing
-  `linalg-detensorize` pass to detensorize input programs. The IR generated by JAX wraps all values
-  in the program in tensors, including scalars, leading to unnecessary memory allocations for
-  programs compiled to CPU via MLIR to LLVM pipeline.
+  @qml.qjit
+  @qml.qnode(dev)
+  def g(x):
+      qml.RX(x, wires=0)
+      return qml.probs(wires=[0])
+  ```
```

Comment:
From Josh (https://github.com/PennyLaneAI/catalyst/pull/1239#discussion_r1820397149):

> @AmintorDusko @erick-xanadu @maliasadi I noticed a (pretty significant?) bug here: ...

Reply:
From offline discussions, it sounds like this was resolved by doing a fresh install.